integration-tests/startup.spec.js

```diff
@@ -16,6 +17,15 @@ const execArgvs = [
 ]
 
 execArgvs.forEach((execArgv) => {
+  // register() was not added until Node 20.6
+  if ((NODE_MAJOR < 20 || (NODE_MAJOR === 20 && NODE_MINOR < 6)) && execArgv.includes('--import')) {
+    return
+  }
+
+  if ((NODE_MAJOR > 20 || (NODE_MAJOR === 20 && NODE_MINOR >= 6)) && execArgv.includes('--loader')) {
+    return
+  }
```

Comment:
Probably would have been clearer to conditionally add to `execArgv` rather than conditionally filtering later here, but I won't block on that. 🤔

Reply:
I agree. I just stole this from `cypress.spec.js` but it'd definitely be clearer if the skip-logic was co-located with the command to be skipped. No rush, can fix.

Reply:
Did some searching and a pattern I see in this repo is conditionally using `describe.skip`. I suppose this is to get more visibility on what is skipped in test logs? With that in mind, what do you think of this?

```js
const execArgvs = [
  {
    execArgv: []
  },
  {
    execArgv: ['--import', 'dd-trace/register.js'],
    skip: NODE_MAJOR < 20 || (NODE_MAJOR === 20 && NODE_MINOR < 6)
  },
  {
    execArgv: ['--loader', 'dd-trace/loader-hook.mjs'],
    skip: NODE_MAJOR > 20 || (NODE_MAJOR === 20 && NODE_MINOR >= 6)
  }
]

execArgvs.forEach(({ execArgv, skip }) => {
  const describe = skip ? globalThis.describe.skip : globalThis.describe

  describe(`startup ${execArgv.join(' ')}`, () => {
```

Reply:
Seems reasonable, though personally I would go for `semver.satisfies(process.versions.node, range)` over those more difficult to read conditions.

Reply:
Neat! Pushed